libs/designsystem/tsconfig.spec.json

```diff
@@ -1,63 +1,64 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
-    "baseUrl": ".",
-    "rootDir": ".",
```

Comment:
To allow setting up an alias for core and include the core files in the designsystem test compilation, we need to remove this to change the rootDir to the base directory of the repo. Otherwise the core files will be "outside the root dir specified for compilation".

Reply:
This in turn affects all the path aliases, but as an added benefit they now have an identical structure to the base.tsconfig.json, and maybe we can do a similar transformation as we do for the jest library down the line, to get rid of these paths?